Components/Transforms/BSplineDeformableTransformWithDiffusion/elxBSplineTransformWithDiffusion.hxx

```diff
@@ -1022,54 +1022,6 @@ BSplineTransformWithDiffusion<TElastix>::WriteToFile(const ParametersType & para
     xl::xout["error"] << excp << std::endl;
   }
 
-  /** Get the GridSize, GridIndex, GridSpacing and
```

Comment:
Please check that `BSplineTransformWithDiffusion::WriteToFile` cannot be entirely removed!

Reply:
indeed